.github/workflows/test.yml

```
@@ -25,10 +27,21 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       groups: ${{ steps.test.outputs.groups }}
+      run_expensive_tests: ${{ steps.envs.outputs.run_expensive_tests }}
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: 'recursive'
+      - id: envs
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" && "${{ github.event.pull_request.labels.*.name }}" == *"run-very-expensive-tests"* ]]; then
+            echo "run_very_expensive_tests=1" >> $GITJUB_OUTPUT
+          elif [[ "${{ github.event_name }}" == "schedule"; then
+            echo "run_very_expensive_tests=1" >> $GITHUB_OUTPUT
+          else
+            echo "run_very_expensive_tests=0" >> $GITJUB_OUTPUT
```

```suggestion
            echo "run_very_expensive_tests=0" >> $GITHUB_OUTPUT
```

_Beware the Jubjub bird_

---

Nice catcj, not sur wjat jappened jere.